src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -31,6 +31,8 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public string BlobNamePrefix { get; set; }
 
+        public ITaskItem[] BlobNames { get; set; }
```

Comment:
In Core-Setup, we only download the blobs we intend to publish - https://github.com/dotnet/core-setup/blob/master/publish/publish.proj#L136